docs/index.md

```diff
@@ -20,7 +20,7 @@ For full information on the spec, plus examples, see [http://jsonapi.org](http:/
 ## Tutorial
 
 Want a tutorial to get started? Read the
-[Laravel tutorial on the *How to JSON:API* website.]((https://howtojsonapi.com/laravel.html))
+[Laravel tutorial on the *How to JSON:API* website.](https://howtojsonapi.com/laravel.html).
```

Comment:
Thanks for spotting this! Can you remove the final full stop as with your change, it would have two! (I prefer the full-stop to be in the square brackets, which is where it currently is.)

Reply:
Ahh, I see that, sorry about that :) 

It is removed now

Reply:
Thanks!